app/views/search/results.html.haml

```diff
@@ -27,6 +27,9 @@
             - unless result._data.nil? or result._data['highlight'].nil?
               %br
               != result._data['highlight']['fulltext'].join("<br />").gsub('<em>','<b>').gsub('</em>','</b>')
+          - elsif result.instance_of?(CollectionsIndex)
+            %b= Collection.model_name
+            %b= link_to([result.title, result.subtitle].compact_blank.join(' / '), collection_path(result.id))
```

Comment:
Probably we may need to add there some tweaks for right-to-left text orientation. Also not sure if subtitle should be displayed at all. I simply did some basic view, we can update later.